Library/Formula/llvm.rb

```diff
@@ -30,11 +30,21 @@ class Llvm < Formula
       sha256 "ae9180466a23acb426d12444d866b266ff2289b266064d362462e44f8d4699f3"
     end
 
+    resource "extra-tools" do
```

Comment:
Please rename this resource back to `clang-extra-tools` and move it to where it used to be. It's easier to understand for everyone if we stick to the names used upstream and I see no good reason to rename/move it.